launchdarkly-server-sdk.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 3.0.0"
 
   spec.add_development_dependency "aws-sdk-dynamodb", "~> 1.57"
+  spec.add_development_dependency "libxml-ruby", "~> 5.0", ">= 5.0.3"
```

Comment:
Later versions of the aws  sdk dynamodb gem seem to require you have this installed. This is only a dev dependency so we can add and remove at will here.